### ISSUE TITLE
type definition for `TweenProps`

### ISSUE THREAD

**Issue by user3587412** [Popmotion/popmotion#238]
Hello, I noticed that the current type definition for `TweenProps` in `./animations/tween/types.d.ts` file was:

```
export declare type TweenProps = {
    from?: number;
    to?: number;
    ...
};
```

However, since `tween`'s `from` and `to` allows for [`Vectors` and colors](https://popmotion.io/api/tween/) as well, I was wondering if the following type would be more appropriate:

```
export declare type Tweenprops = {
    from?: number | number[] | { [name: string]: number | string } | string;
    to?: number | number[] | { [name: string]: number | string } | string;
    ...
}
```

Thanks!

**Comment by InventingWithMonster** [comment]
Hey man, thanks for pointing this out! You are correct. I'll try and fix asap

**Comment by InventingWithMonster** [comment]
@user3587412 This has been fixed in `8.1.16`

**Comment by user3587412** [comment]
Thank you!
